src/stories/Library/search-result-item/SearchResultItem.tsx

```diff
@@ -39,7 +39,9 @@ export const SearchResultItem = ({
           )}
         </div>
 
-        <h2 className="search-result-item__title text-header-h4">{title}</h2>
+        <h2 className="search-result-item__title text-header-h4">
+          <a href="">{title}</a>
```

Comment:
Please consider why we need implement a separate `<a>` tag which if implemented would link to the same page as the outer link for the item.

If the only intention is to add underline to the title on hover could we implement this behavior directly on the `.search-result-item__title` class?

Reply:
It was not possible for me to change the styling of the a tag within the .search-result-item__title because the user agent stylesheet for the a tag will overwrite any cascading styles from the .search-result-item__title